facades/PC/src/main/java/org/terasology/engine/Terasology.java

```diff
@@ -42,7 +41,7 @@
 import org.terasology.rendering.nui.layers.mainMenu.savedGames.GameInfo;
 import org.terasology.rendering.nui.layers.mainMenu.savedGames.GameProvider;
 
-import java.awt.GraphicsEnvironment;
+import java.awt.*;
```

Comment:
No wildcard imports please, especially if only one class is actually needed from that package.